src/api/app/models/package.rb

```diff
@@ -165,6 +165,8 @@ def self.striping_multibuild_suffix(name)
   # use_source: false to skip check for sourceaccess permissions
   # function returns a nil object in case the package is on remote instance
   def self.get_by_project_and_name(project, package, opts = {})
+    raise UnknownObjectError, "#{project}/#{package}" unless (project && package)
```

Comment:
I do not like this too much, it would hide problems when the method is called wrong.

IMHO you need to strip the multibuild suffixes before instead.